connectors/cli.py

```diff
@@ -90,11 +90,14 @@ def _parser():
 
 def run(args):
     """Runner"""
+
+    # load config
+    load_config(args.config_file)
```

Comment:
How do you feel about changing it to `config = load_config(args.config_file)` so that it's not globally accessible?

Reply:
Ah, I see @tarekziade left the same comment. I'm not a fan of singleton config - is there a need for it now for certain task to finish?

Globally accessible config often promotes unhealthy coding styles IMO.

Reply:
Hi @artem-shelkovnikov , I implemented `config` the same way as [logger](https://github.com/elastic/connectors-python/blob/e342bf0426b00ac46a5d4a3d4f20c5827ca29769/connectors/logger.py#L16).

We had a global variable `logger` because we want to use the same logger instance to log messages across the whole project. This is the same for `config`, which we don't want to load it multiple times.

What you suggested is to introduce a `load_config` function (basically just a wrapper of `EnvYAML`), and use it every time when we need config.

Reply:
> What you suggested is to introduce a load_config function (basically just a wrapper of EnvYAML), and use it every time when we need config.

No, the suggestion is to keep it the same way it's done now - it's loaded once by the CLI/entry point class before the services start and then the config is passed to the services. The services then pass the config to the other classes/methods that need config.